exchanges/subscription/subscription.go

```diff
@@ -140,8 +140,8 @@ func (s *Subscription) Clone() *Subscription {
 		Pairs:            s.Pairs,
```

Comment:
```suggestion
```
Removes the double-assign and potentially what allowed us to get confused.
Do the same for `Params` too